### PR TITLE
Adapt to new PaperButtonBehavior

### PR DESCRIPTION
**Pull request by sorvell**
Fixes https://github.com/PolymerElements/paper-button/issues/48

### CODE REVIEW

File: paper-fab.html [hunk 1]
```diff
@@ -98,27 +100,15 @@
...
+    :host(.keyboard-focus) {
       background: var(--paper-fab-keyboard-focus-background, --paper-pink-900);
     }
   </style>
-  <template>
-    <paper-ripple></paper-ripple>
-    <paper-material class$="[[_computeContentClass(receivedFocusFromKeyboard)]]" elevation="[[_elevation]]" animated>
-      <iron-icon id="icon" src="[[src]]" icon="[[icon]]"></iron-icon>
-    </paper-material>
-  </template>
```
Comment (cdata) [inline paper-fab.html]: :trophy:

**Comment by sorvell** [comment]
@cdata PTAL

File: demo/index.html [hunk 2]
```diff
@@ -63,33 +63,33 @@
...
-        <paper-fab icon="arrow-forward" title="arrow-forward" tabindex="0" class="blue"></paper-fab>
-        <paper-fab icon="create" title="create" tabindex="0" class="red"></paper-fab>
-        <paper-fab icon="favorite" title="heart" tabindex="0" class="orange"></paper-fab>
-        <paper-fab mini icon="done" title="done" tabindex="0" class="green"></paper-fab>
-        <paper-fab mini icon="reply" title="reply" tabindex="0" class="blue"></paper-fab>
+        <paper-fab icon="arrow-forward" title="arrow-forward" class="blue"></paper-fab>
+        <paper-fab icon="create" title="create" class="red"></paper-fab>
+        <paper-fab icon="favorite" title="heart" class="orange"></paper-fab>
+        <paper-fab mini icon="done" title="done" class="green"></paper-fab>
+        <paper-fab mini icon="reply" title="reply" class="blue"></paper-fab>
```
Comment (cdata) [inline demo/index.html]: Should we add a `noink` demo?

Comment (cdata) [inline demo/index.html]: Resolved!

File: demo/index.html [hunk 3]
```diff
@@ -63,33 +63,44 @@
...
+    </div>
+
+    <div>
+      <h4>Noink</h4>
+      <div class="horizontal-section">
+        <paper-fab noink icon="arrow-forward" title="arrow-forward"></paper-fab>
+        <paper-fab noink icon="create" title="create"></paper-fab>
+        <paper-fab noink icon="favorite" title="heart"></paper-fab>
+        <paper-fab noink mini icon="done" title="done"></paper-fab>
+        <paper-fab noink mini icon="reply" title="reply"></paper-fab>
```
Comment (cdata) [inline demo/index.html]: :lock_with_ink_pen:

**Pull request merged**